packages/core/src/api/joints/mutant/injectLiveMutantReferences.ts

```diff
@@ -56,15 +54,15 @@ export async function injectLiveMutantReferences(props: {
       }
 
       const args = unpackToRawMutantArgs(mutantCell.cellOutput.type!.args);
-      const minPayment = args.minPayment !== void 0 ? BI.from(10).pow(args.minPayment) : BI.from(0);
+      const minPayment = args.minPayment !== void 0 ? args.minPayment : BI.from(0);
```

Comment:
why use shannon directly

Reply:
prior implementation is in shannon, we can turn it to human-readable ckb in our high-level API